dev-cpp/frugally-deep/frugally-deep-0.15.30.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Header-only library for using Keras models in C++"
+HOMEPAGE="https://github.com/Dobiasd/frugally-deep"
+SRC_URI="https://github.com/Dobiasd/frugally-deep/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+IUSE="double"
+
+DEPEND="
+	dev-cpp/functional-plus
+	dev-cpp/eigen
+	dev-cpp/nlohmann_json
+"
+
+RDEPEND="${DEPEND}"
+
+src_configure() {
+	local mycmakeargs=(
+		-DFDEEP_USE_DOUBLE=$(usex double)
```

Comment:
I wonder if we really need USE=double here given nothing depends on it. If it's useful though, that's fine, it's just that historically, flags like this (double and such) have ended up breaking things when on/off, or wose, breaking ABI.

Reply:
Removed `src_configure` completely. It makes no sense for header-only library (it only would make sense if somebody would include frugally-deep with `add_subdirectory`)